### ISSUE TITLE
Fetching metadata error

### ISSUE THREAD

**Issue by anon113** [Pewpews/happypanda#135]
After selecting some galleries and click "Download metadata for selected" it doesn't download metadata, just show progress ring.

https://dl.dropbox.com/s/r3o2r4pjecxofe9/happypanda_debug4.log

**Comment by JacobDNN** [comment]
Same for me.
[happypanda_debug.zip](https://github.com/Pewpews/happypanda/files/150546/happypanda_debug.zip)

**Comment by JacobDNN** [comment]
Also, looks like the same error sometimes happens when adding galleries too. Or maybe it's different.
Here's the gallery 
https://mega.nz/#!dgAgGDyY!8efnX_1V5Q1RD8kXN8b9Y_zg0U_vrbAdPhFsqAxWDE4

```
29-02 15:22 INFO   gallerydb Recevied gallery: b'E:/123/[Hiroshi Kawamoto] Gougeki Cartoonists Daisakusen.zip'
29-02 15:22 INFO   utils  Getting image from archive
29-02 15:22 CRITICAL app      File "c:\python\32-bit\3.4\lib\threading.py", line 889, in _bootstrap
  File "c:\python\32-bit\3.4\lib\threading.py", line 930, in _bootstrap_inner

29-02 15:22 CRITICAL app    <class 'AttributeError'>: 'NoneType' object has no attribute 'write'
```

[happypanda_debug.txt](https://github.com/Pewpews/happypanda/files/150793/happypanda_debug.txt)

**Comment by twiddli** [comment]
Fixed in v0.29
